composer.json

```diff
@@ -12,9 +12,12 @@
 	"autoload": {
 		"files": [
 			"load.php"
-		]
+		],
+		"classmap": [ "inc/", "inc/stream/" ]
 	},
 	"require": {
-
+		"humanmade/stream": "dev-build",
```

Comment:
Should I review these first? Can set up the build step -> tag release process on circleci if you need

Reply:
Because it's a fork of stream, the tags have to be deleted, and then re-tagged on the build branch, which I did this time manually. If it's possible to set that up with CI, happy for that, but don't have much time to give to that atm